pkg/scm/scm.go

```diff
@@ -46,5 +47,9 @@ func DownloaderForSource(s string, forceCopy bool) (build.Downloader, string, er
 		return &git.Clone{g, util.NewFileSystem()}, s, nil
 	}
 
+	if len(s) == 0 {
+		return &empty.Noop{}, s, nil
+	}
```

Comment:
should this be at the top of this function instead of the bottom?  not sure what git.ParseFile() does with an empty string, but probably best not to even call it.